### PR TITLE
Fix #1886

### LINKED ISSUE #1886: Function resolving when searching for calls
Hi!

It would be great that when searching, for example, for calls with /c call, the destination address would be resolved as it's resolved in the disassembly view. 

For example, we would get an output like the following:
```
f hit_0 @ 0x100052f1   # 6: call dword [0x10016010]
f hit_1 @ 0x100052fe   # 6: call dword [0x10016008]
f hit_2 @ 0x1000532a   # 2: call ebx
f hit_3 @ 0x10005341   # 3: call dword [eax-0x1]
f hit_4 @ 0x10005343   # 2: call edi
```

In the disassembly view, the call to 0x10016010 would be resolved and the function name would be shown. But it's not when searching.

Edit: Here's an asciinema showing the problem and a workaround:
https://asciinema.org/a/2o4bpkbq1zfypjx62iqklo2ux

The binary used is the Lab06-02.exe from Practical Malware Analysis. It can be downloaded from https://practicalmalwareanalysis.com/labs/

Thanks for your great work!

### PR DESCRIPTION
**Pull request by rlaemmert**

### CODE REVIEW

File: libr/core/cmd_search.c [hunk 1]
```diff
@@ -1711,8 +1712,15 @@ static void do_asm_search(RCore *core, struct search_parameters *param, const ch
 							searchprefix, kwidx, count, hit->addr);
 					break;
 				default:
-					r_cons_printf ("0x%08"PFMT64x"   # %i: %s\n",
-							hit->addr, hit->len, hit->code);
+					if (filter) {
+						char tmp[128] = {};
```
Comment (radare) [inline libr/core/cmd_search.c]: fill it with ezros with {0}

File: libr/core/cmd_search.c [hunk 2]
```diff
@@ -1711,8 +1712,15 @@ static void do_asm_search(RCore *core, struct search_parameters *param, const ch
...
+						r_parse_filter (core->parser, core->flags, hit->code, tmp, sizeof (tmp), core->print->big_endian);
+						r_cons_printf ("0x%08"PFMT64x"   # %i: %s\n",
```
Comment (radare) [inline libr/core/cmd_search.c]: if parse.filter fails the string will not be null terminated

**Comment by radare** [comment]
travis seems unhappy

**Comment by radare** [comment]
..... why do you merge stuff that is red?

**Comment by Maijin** [comment]
@radare Because I fixed the tests :)

**Comment by radare** [comment]
ok

**Pull request merged**